json-schema-form.d.ts

```diff
@@ -1,8 +1,9 @@
+import { WARNING_TYPES } from './src/modify';
```

Comment:
I would say this file shouldn't do imports, unless it's importing type utils. In that case I would prefer that `MODIFY_WARNING_TYPES` would be exported from here – suggesting a new name since the warnings are only related to `modify` call.

Reply:
Nevermind, this is not possible in declaration files. You can still create the type with the union

```
type WarningType = "FIELD_TO_CHANGE_NOT_FOUND" | "ORDER_MISSING_FIELDS" | "FIELD_TO_CREATE_EXISTS" | "PICK_MISSED_FIELD"
```

Reply:
Hmmmm it does not work. I believe we cannot write JS in .d.ts files [image](https://github.com/user-attachments/assets/99c832b5-bae9-4571-8b1f-b2dd7c427edf). Perhaps the best is to duplicate in both places for now?

Reply:
Yeah. And I realized my solution with import didn't work either. It says the type is `any` [screenshot](https://github.com/user-attachments/assets/e1360c6e-82e1-476a-a6f3-d6b0f2f088d6).

So I decided to duplicate the strings in both files with a code comment. Ready for a new review